compose/src/main/java/com/bottlerocketstudios/compose/devoptions/DevOptions.kt

```diff
@@ -0,0 +1,308 @@
+}
+
+@Composable
+private fun AppInfoCard(state: DevOptionsState) {
+    CardTitle(cardTitle = "App Info")
+    TitleValueRow(title = "Version Name", entryValue = state.appVersionName)
+    TitleValueRow(title = "Version Code", entryValue = state.appVersionCode)
+    TitleValueRow(title = "Application ID", entryValue = state.appId)
+    TitleValueRow(title = "Build Identifier", entryValue = state.buildIdentifier)
+}
+
+@Composable
+private fun MiscFunctionalityCard(onForceCrashCtaClick: () -> Unit) {
+    CardTitle(cardTitle = "Misc Functionality")
+    PrimaryButton(
+        buttonText = "FORCE CRASH",
+        onClick = onForceCrashCtaClick,
+        modifier = Modifier.fillMaxWidth()
+    )
+}
+
+@Composable
+private fun CardDivider(dividerColor: Color) {
+    Divider(
+        color = dividerColor,
+        thickness = ArchitectureDemoTheme.dimens.grid_0_25,
+        modifier = Modifier.padding(
+            top = ArchitectureDemoTheme.dimens.grid_1_5,
+            bottom = ArchitectureDemoTheme.dimens.grid_1_5
+        )
+    )
+}
+
+@Composable
+private fun CardTitle(cardTitle: String) {
+    Text(
+        cardTitle,
+        style = MaterialTheme.typography.h3.normal(),
+        modifier = Modifier
+            .padding(bottom = ArchitectureDemoTheme.dimens.grid_1)
+            .fillMaxWidth()
+    )
+}
+
+@Composable
+private fun TitleValueRow(title: String, entryValue: String) {
+    Text(
+        title,
+        style = MaterialTheme.typography.h3.bold(),
+        modifier = Modifier
+            .padding(
+                top = ArchitectureDemoTheme.dimens.grid_1,
+                bottom = ArchitectureDemoTheme.dimens.grid_1
+            )
+            .wrapContentHeight()
+            .fillMaxWidth()
+    )
+    Text(
+        entryValue,
+        style = MaterialTheme.typography.h3.light(),
+        modifier = Modifier
+            .wrapContentHeight()
+            .fillMaxWidth()
+    )
+}
+
+@Composable
+private fun CardLayout(content: @Composable () -> Unit) {
+    Card(elevation = ArchitectureDemoTheme.dimens.plane_3) {
+        Column(
+            modifier = Modifier
+                .padding(
+                    start = ArchitectureDemoTheme.dimens.grid_1,
+                    end = ArchitectureDemoTheme.dimens.grid_1,
+                    top = ArchitectureDemoTheme.dimens.grid_1_5,
+                    bottom = ArchitectureDemoTheme.dimens.grid_1_5
+                )
+        ) {
+            content()
+        }
+    }
+}
+
+// TODO this needs to be update when the next stable release comes out. New Menus
+@Composable
+@Suppress("LongMethod")
+private fun DropdownEnvMenu(state: DevOptionsState) {
+    var expanded by remember { mutableStateOf(false) }
+    val items = state.environmentNames.value
+    Box(
+        modifier = Modifier
+            .fillMaxWidth()
+            .wrapContentSize(Alignment.TopStart)
+    ) {
+        items[state.environmentSpinnerPosition.value].let {
+            Row(
```

Comment:
Feels like a composable here to create the item from items would make this small enough to remove suppress for "LongMethod", passing in position and expanded?